<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestProjectClass</class>
 <widget class="QMainWindow" name="QtTestProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Create Point</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnCreatePoint">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Create Line</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnCreateLine">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Create Polygon</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnCreatePolygon">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabPoint">
       <attribute name="title">
        <string>Point</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabLine">
       <attribute name="title">
        <string>Line</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabPolygon">
       <attribute name="title">
        <string>Polygon</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtTestProject.qrc"/>
 </resources>
 <connections/>
</ui>
